internal/proxy/keep_active_interceptor.go

```diff
@@ -0,0 +1,41 @@
+package proxy
+
+import (
+	"context"
+	"fmt"
+	"strconv"
+
+	"google.golang.org/grpc/metadata"
+
+	"google.golang.org/grpc"
+)
+
+func getIdentifierFromContext(ctx context.Context) (int64, error) {
+	md, ok := metadata.FromIncomingContext(ctx)
+	if !ok {
+		return 0, fmt.Errorf("fail to get metadata from the context")
+	}
+	identifierContent, ok := md["identifier"]
```

Comment:
make this Const definition. just like util.HeaderAuthorize in pkg/util/constant.go

Reply:
If you add timeout logic here, there is no need to add timeout logic in keepActive.